docs/api/useFirestoreConnect.md

```diff
@@ -26,39 +26,42 @@ _Basic_
 
 ```javascript
 import React from 'react'
-import { map } from 'lodash'
 import { useSelector } from 'react-redux'
-import { useFirebaseConnect } from 'react-redux-firebase'
+import { useFirestoreConnect } from 'react-redux-firebase'
```

Comment:
These docs are actually generated by the [comments within the source code](https://github.com/prescottprue/react-redux-firebase/blob/master/src/useFirestoreConnect.js/#L21), so we will want to update them there

Reply:
I see...

It would be really nice if there was a section on contributing to the docs in the repo somewhere (unless I've missed it).  It would save people from running into this issue in the future.

I'd do it myself but I'm sure I'll miss something else important @prescottprue